### DISCUSSION TITLE
Getting data from Ultralytics app

### DISCUSSION THREAD

**Discussion by tininha94** [ultralytics/hub#40] [Q&A]
Hi, I'm working on a academic project to identify problems in cans through the phone camara, and I need some help with the data export. My project was made in the ultralytics hub with an customized dataset and we access our project through the ultralytics app for us to be able to use the phone camara. But now I have to access the result shown in the app. The first idea was to integrate the ultralytics app with a react native app, so when we press a button in the react native app the percentage value was storaged, but I couldnt find any way to do that.

I was wonding if there is any way to export the percentage value shown in the ultralytics app in real time.

**Comment by glenn-jocher** [comment]
@tininha94 thanks for asking! There's no way to extract data at the moment, but we will take your feedback into consideration when planning our product roadmap.
